easybuild/easyconfigs/n/ncregrid/ncregrid-1.4b-goolf-1.4.10.eb

```diff
@@ -0,0 +1,47 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2014 The Cyprus Institute
+# Authors:: Thekla Loizou <t.loizou@cyi.ac.cy>
+# License:: MIT/GPL
+#
+##
+easyblock = 'ConfigureMake'
+
+name = 'ncregrid'
+version = '1.4b'
+
+homepage = 'http://www.pa.op.dlr.de/~PatrickJoeckel/ncregrid/'
+description = """ncregrid is a tool (written in FORTRAN-90) for data
+ transfer of gridded 2- and 3-dimensional (spatial) geophysical/geochemical
+ scalar fields between grids of different resolutions"""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+toolchainopts = {'optarch': True, 'usempi': True, 'pic': True}
+
+source_urls = ['http://www.pa.op.dlr.de/~PatrickJoeckel/ncregrid/']
+sources = ['%(name)s.tar.Z']
```

Comment:
the very minimal would be to add `osdependencies = ['compress']` here

but I prefer fixing this via a build dep instead, so yes, then it requires fixing https://github.com/hpcugent/easybuild-framework/issues/1376